proto/umee/historacle/v1/historacle.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package umee.historacle.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "umee/oracle/v1/oracle.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/historacle/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// Params defines the parameters for the historacle module.
+message Params {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_stringer) = false;
+  uint64 stamp_period                 = 1;
```

Comment:
Maybe add a comment to define what stamp_period and prune_period are?

Reply:
15b70efb8230123a7e58873b98da247fec2872ea